styles/about.css

```diff
@@ -43,3 +50,48 @@
   color: #000;
   margin-top: 30px;
 }
+
+.title-founder {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  margin-bottom: 40px;
+  margin-top: 50px;
+  margin-left: 70px;
+}
+
+.container-founder {
+  display: flex;
+  margin-left: 70px;
+  align-items: center;
+} 
+
+.img-founder-jason {
+  width: 150px;
+  height: 150px; 
+}
```

Comment:
It seems that `.img-founder-jason` and `.img-founder-maria` are very similar. How about we make one css class for both images:

`.img-founder {
  width: 150px;
  height: 150px; 
  margin-right: 90px;
}`

Reply:
the left margin is different, one is 70px and the other 90px, what do you advise?
Do I leave a single one or keep both classes?